benchmarks/bench_covertype.py

```diff
@@ -102,7 +102,7 @@ def load_data(dtype=np.float32, order='C', random_state=13):
     'ExtraTrees': ExtraTreesClassifier(n_estimators=20),
     'RandomForest': RandomForestClassifier(n_estimators=20),
     'CART': DecisionTreeClassifier(min_samples_split=5),
-    'SGD': SGDClassifier(alpha=0.001, n_iter=2),
+    'SGD': SGDClassifier(alpha=0.001, max_iter=2, tol=0),
```

Comment:
The tol is there to make sure that the iteration goes all the way to the end?

If so, it might be a good idea to add a comment.

Also, "0" is a bit of an extreme value.